.github/workflows/buildExamples.yml

```diff
@@ -36,3 +36,8 @@ jobs:
 
       - name: Build examples
         run: yarn workspaces foreach -p run build-storybook
+
+      - name: Publish examples to Chromatic
+        run: yarn workspaces foreach -p run publish-chromatic
```

Comment:
I think you'll need to do something like `CHROMATIC_PROJECT_TOKEN={secrets.CHROMATIC_PROJECT_TOKEN} yarn workspaces foreach ...`

Reply:
Fixed now! Looks like it's failing on something else 🤷 strange 400 error from Chromatic.

Reply:
I've seen errors like that before when I had a story that would build, but had a runtime error.  I'll try to poke around a little and see if I can figure out what's going on.  

Also, I wonder if we'll need to create a "main" storybook that composes the others together so that we have a single storybook for chromatic.  Would you like me to push some commits onto this PR to test it out?

Reply:
All the stories seem like they work fine: https://61ccd7616b9029003af2a695-fbkrmjxuyt.chromatic.com/.  I'm not sure what's going on, maybe worth sending the error log to the chromatic chat?  I've found the team to be very helpful when I've had similar issues in the past.